src/ipynb/spark_ml.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    geo_pd = geo_pd[["zone_id", "long", "lat"]]
</pre><p>Is this correct? It looks like you're just creating "long" and "lat" in the previous step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>The initial schema of geo_pd is as follows: <code>[zone_id, zone_name, borough, zone_geom]</code> .</p><p>Following the execution of line 13, the schema changed to: <code>[zone_id, zone_name, borough, zone_geom, long, lat]</code> . </p><p>By line #16, certain columns such as <code>zone_name, borough, and zone_geom</code>  are removed.</p><p> </p><p>If we don't remove these columns, <code>taxi_zone_df</code> ends up having duplicate columns. This happens because <code>taxi_zone_df</code>   two sets of DataFrames (<code>geo_spark_df</code> and <code>taxi_df</code>) twice, so without dropping these columns results in an error.</p>